change/@fluentui-react-native-dependency-profiles-9b7c2763-ecdc-43b6-b097-c5ec8bda8d9c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
These files would cause another bump right? Is that intended?

Reply:
That's right - I'm not sure that we need another bump but it's what we've done in the past. Otherwise, the PR build will complain there are no change files present.

Reply:
I guess I could mark it as "no change" instead of "patch" so it doesn't actually bump?